.gitignore

```diff
@@ -72,6 +75,7 @@ python
 Downloads
 src-extern
 notes
+.pylintrc
```

Comment:
I'm not terribly familiar with this file, but it seems like something that should be in source control?